app/src/main/java/org/simple/clinic/patient/recent/RecentPatientItem.kt

```diff
@@ -0,0 +1,24 @@
+package org.simple.clinic.patient.recent
+
+import com.xwray.groupie.Item
+import com.xwray.groupie.ViewHolder
+import kotlinx.android.synthetic.main.recent_patient_item_view.view.*
+import org.simple.clinic.R
+import org.simple.clinic.patient.Gender
+
+class RecentPatientItem(
+    val title: String,
```

Comment:
Is this the patient name?
Can you rename this to `patientName` or even `name`

Reply:
This includes patient name and age. Let's keep it as is?

Reply:
I've split name and age into two different fields.